perception/lidar_centerpoint/launch/lidar_centerpoint.launch.xml

```diff
@@ -8,6 +8,7 @@
   <arg name="class_remapper_param_path" default="$(find-pkg-share lidar_centerpoint)/config/detection_class_remapper.param.yaml"/>
   <arg name="score_threshold" default="0.35"/>
   <arg name="has_twist" default="false"/>
+  <arg name="build_only" default="false" description=""/>
```

Comment:
Is it necessary to add empty description?

Reply:
added at [a8a7912](https://github.com/autowarefoundation/autoware.universe/pull/2677/commits/a8a791266ab8ccdd98008a8f54ad65f0f9e1128f).